Release-Plan/Documentation-Release-Plan.md

```diff
@@ -0,0 +1,30 @@
+# CFDE Published Documentation Release Plan
+
+This is the release plan for the [CFDE Published Documentation website](https://cfde-published-documentation.readthedocs-hosted.com/en/latest/).
+
+## Release Documentation
+
+The Github repo for the published documentation website will host a release section and associated tag information.
+
+For documentation version numbering system we will follow the release semantics with Major and Minor numbering. Thus our release tag would have the format: **`Major.Minor`**
+
+The current version of the published documentation is at `v1.0`. Any subsequent updates via PRs to the repo will be documented and can be released as either a `Minor` or `Major` release depending on the scope of the content. For example updates to an existing use case might be might be considered a minor update with tag as **`v1.1`**.
```

Comment:
to an existing document maybe? not use case

Reply:
I have made the corrections.